style.css

```diff
@@ -0,0 +1,80 @@
+.box {
+  margin: 24px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  flex-grow: 4;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+span {
+  display: inline-block;
+}
+
+span + span {
+  margin-left: 221px;
+}
+
+.content {
+  flex-grow: 6;
+}
+
+.title {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 52px;
+  color: #344563;
+}
+
+.paragraph {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.connect {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.network-icons {
+  display: inline-block;
+  column-gap: 300px;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  height: 718px;
+  width: 375px;
+  outline: 2px solid black;
+  background-size: 100% 80%;
+  background-image: url("img/background.jpg");
+  background-repeat: no-repeat;
+  background-position: 0 110px;
+}
```

Comment:
- [ ] According to the requirements, You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)., so Kindly remove the `outline` from the container 👌

![image](https://user-images.githubusercontent.com/45231276/191668286-f7d787c3-ad3e-4641-b68e-b7e18d6567e2.png)